packages/styled-components/components/logo/index.js

```diff
@@ -36,6 +36,9 @@ const Logo = (props) => {
     <LogoWrapper {...standardProps}>
       <LogoLink
         as={Link}
+        gtmAction="Homepage"
```

Comment:
Are we sure this should always be "Homepage?"

Reply:
@joemcgill I guess not – maybe we should move these to the `irving/site-link` definition in WP Irving? What do you think?

Reply:
I'd like us to consider making `gtm` an object with each of these properties, and I also agree that setting the defaults in WP Irving is ideal.

Reply:
@jameswburke @joemcgill Are you two open to getting this out as-is and then I will follow up to make any GTM values in WP Irving or Irving configurable from WP Irving?

Other question: Do we have any prior art for this? Or suggestions on existing features to look at for inspiration on approach?

Reply:
@davisshaver, I confirmed with @joemcgill that we'd like to see this updated since it's not blocking anything and we need to go through the release process. We can both swarm with you today to help get it over the finish line.